defold-spine/commonsrc/vertices.cpp

```diff
@@ -295,4 +298,30 @@ uint32_t GenerateVertexData(dmArray<SpineVertex>& vertex_buffer, const spSkeleto
     return vcount;
 }
 
+void MergeDrawDescs(const dmArray<SpineDrawDesc>& src, dmArray<SpineDrawDesc>& dst)
```

Comment:
I think we could use a single array here, but it's a lot easier to read this function when we have a secondary buffer.